templates/detail/submission.html

```diff
@@ -87,17 +87,10 @@ <h1 id="detail-title" class="pad-left pad-right">${query['title']} <i>by</i> <a
       $if cover:
         <img src="${cover['display_url']}" alt="${query['title']}" />
       <div id="detail_submission_media" style="display:inline-block;width:500px;">
-        <div id="multimedia"></div>
-        <script type="text/javascript">
-        jwplayer("multimedia").setup({
-          flashplayer: "${CDNIFY('/static/mediaplayer/player.swf')}",
-          controlbar: "bottom",
-          file: $:{INLINE_JSON(submission['display_url'])},
-          icons: false,
-          width: 500,
-          height: 24
-        });
-        </script>
+        <audio controls>
+          <source type="audio/mpeg" src="${submission['display_url']}">
```

Comment:
You can just use `<audio type="audio/mpeg" src="…" controls>` if there’s only one source.